src/calliope/backend/__init__.py

```diff
@@ -1 +1,32 @@
 """Callipe's backend module."""
+
+from xarray import Dataset
```

Comment:
I prefer keeping the source library where possible, so `import xarray as xr` and then `xr.Dataset`. Makes it easier to track what is an calliope method/class and what isn't.